runtime/build.gradle.kts

```diff
@@ -41,6 +41,8 @@ kotlin {
 
     sourceSets {
         all {
+            languageSettings.enableLanguageFeature("InlineClasses")
+
```

Comment:
👾

Reply:
This was actually intentional to provide some visual separation between the `enableLanguageFeature` configs and the `useExperimentalAnnotation` configs.